lib/src/util.rs

```diff
@@ -81,6 +85,48 @@ pub fn create_yaml(folder_path: &str, file_name: &str) -> String {
     file_path
 }
 
+#[derive(Serialize, Deserialize)]
+struct DAGSchedulerResultInfo {
+    schedule_length: i32,
+    period_factor: f32,
+    result: bool,
+}
+
+pub fn dump_dag_scheduler_result_to_yaml(
+    file_path: &str,
+    schedule_length: i32,
+    period_factor: f32,
+    result: bool,
+) {
+    let result_info = DAGSchedulerResultInfo {
+        schedule_length,
+        period_factor,
+        result,
+    };
+    dump_struct(file_path, &result_info);
+}
+
+#[derive(Serialize, Deserialize)]
+struct DAGSetSchedulerResultInfo {
+    schedule_length: i32,
+    hyper_period: i32,
```

Comment:
These are not important for multi-DAG scheduling and should be removed. :pray: 

```suggestion
```

Reply:
Deleted.